src/main/java/com/programmerdan/minecraft/simpleadminhacks/hacks/GameTuning.java

```diff
@@ -139,4 +141,16 @@ public void chunkLimitsExploitExtend(BlockPistonExtendEvent event) {}
 	// TODO
 	@EventHandler(priority = EventPriority.LOWEST, ignoreCancelled=true)
 	public void chunkLimitsExploitRetract(BlockPistonRetractEvent event) {}
+
+	@SuppressWarnings("deprecation")
+	@EventHandler(priority = EventPriority.LOWEST, ignoreCancelled = true)
+	public void bedRClickToSetSpawn(PlayerInteractEvent event) {
+		if (event.getAction() != Action.RIGHT_CLICK_BLOCK || event.getClickedBlock().getType() != Material.BED_BLOCK ||
+				!config.areDaytimeBedsEnabled()) {
+			return;
+		}
+
+		event.getPlayer().setBedSpawnLocation(event.getClickedBlock().getLocation());

```

Comment:
setBedSpawnLocation works like vanilla. It spawns the player near the location, not exactly at the location.